src/libs/ReportUtils.js

```diff
@@ -2202,6 +2202,9 @@ function getIOUReportActionMessage(iouReportID, type, total, comment, currency,
         case CONST.REPORT.ACTIONS.TYPE.APPROVED:
             iouMessage = `approved ${amount}`;
             break;
+        case CONST.REPORT.ACTIONS.TYPE.SUBMITTED:
+            iouMessage = `submitted ${amount}`;
```

Comment:
This line introduced bug here: https://github.com/Expensify/App/issues/38579
where the optimistic data from the client differs from what is returned by the backend.